### PR TITLE
Adds frxETH 

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
# Adds frxETH on EigenLayer

This PR adds the liquidity pool of frxETH on eigenLayer to treasury

Fixes https://github.com/EveripediaNetwork/issues/issues/2589

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -14,7 +14,16 @@ const nextConfig = {
...
+  webpack(config) {
+    config.resolve.fallback = {
+      fs: false,
+      net: false,
+      tls: false,
+      'react-native': false,
+    }
+    config.externals.push('pino-pretty', 'lokijs', 'encoding')
+    config.optimization.moduleIds = 'named'
+    config.optimization.runtimeChunk = 'single'
```
Comment (kesar) [inline next.config.js:26]: next time avoid config changes unrelated to the PR. this PR needed to urgently get merged and this config changes makes the PR with more chances of introducing unexpected bugs

File: src/data/treasury-data.ts [hunk 2]
```diff
@@ -110,6 +110,18 @@ export const TOKENS: TokensType = {
     name: 'stkCvxFPIS',
     address: '0xfa87db3eaa93b7293021e38416650d2e666bc483',
   },
+  eigenlayer_deposit2: {
```
Comment (kesar) [inline src/data/treasury-data.ts:113]: next time lets do better at naming

File: src/data/treasury-data.ts [hunk 3]
```diff
@@ -110,6 +110,18 @@ export const TOKENS: TokensType = {
...
+    id: 'eigenlayer_deposit2',
+    name: 'frxETH',
+    image: '/images/frxEth.svg',
+    address: '0x39053d51b77dc0d36036fc1fcc8cb819df8ef37a',
+  },
+  eigenlayer_yield: {
+    id: 'eigenlayer_yield',
+    name: 'frxETH',
+    image: '/images/frxEth.svg',
```
Comment (SidharthK2) [inline src/data/treasury-data.ts:122]: `/images/frxETH.svg`

File: next.config.js (hunk 1 above)
Comment (Adebesin-Cell) [reply next.config.js:26]: Yes ser. It was some lines to stop some weird terminal logs caused by pino pretty :))

File: src/data/treasury-data.ts (hunk 2 above)
Comment (Adebesin-Cell) [reply src/data/treasury-data.ts:113]: Noted sir.

File: src/data/treasury-data.ts (hunk 3 above)
Comment (Adebesin-Cell) [reply src/data/treasury-data.ts:122]: Got it, I’ll fix it.

**Pull request merged**